solidity/random-beacon/contracts/libraries/Relay.sol

```diff
@@ -70,7 +82,15 @@ library Relay {
         bytes previousEntry
     );
 
-    event RelayEntrySubmitted(uint256 indexed requestId, bytes entry);
+    event RelayEntrySubmitted(
+        uint256 indexed requestId,
+        bytes entry,
+        uint256 submissionBlock,
+        uint256 eligibilityDelay,
+        uint256 requestStartBlock,
+        uint256 submitterIndex,
+        uint32[] groupMembers
```

Comment:
The cost of emitting event with this field will be significant. Given that the function reporting inactivity can only be called before the next request comes in, I think we may not need all fields in this event. Some information is available on-chain.

Reply:
Fetching that on-chain is even more expensive.

Reply:
> Given that the function reporting inactivity can only be called before the next request comes in

That's not true. According to what we discussed, this function can be called until the next relay entry is submitted, not just the request.

Reply:
> Fetching that on-chain is even more expensive.

I was thinking about fetching it off-chain and providing to the notify function but I haven't had a chance to look at the rest of the code so unsure if this is feasible and what are the consequences.

Reply:
Yeah in that case I think we can just emit `groupId`. I'll check that.